docs/src/series.md

```diff
@@ -482,8 +482,38 @@ julia> d = inv(b)
 
 ```
 
+### Derivative and integral
+
+```@docs
+derivative(a::RelSeriesElem)
+```
+
+```@docs
+integral(a::RelSeriesElem)
+```
+
+**Examples**
+
+```jldoctest
+julia> R, x = PowerSeriesRing(QQ, 10, "x")
+(Univariate power series ring in x over Rationals, x + O(x^11))
+
+julia> f = 2 + x + 3x^3
+2 + x + 3*x^3 + O(x^10)
+
+julia> derivative(f)
+1 + 9*x^2 + O(x^9)
+
+julia> integral(f)
+2*x + 1//2*x^2 + 3//4*x^4 + O(x^11)
```

Comment:
How about moving these kinds of examples into the respective docstrings, then they are also available in the REPL help. Julia provides many such examples in the docstrings, and I think that's very beneficial for many users.

Reply:
Done.